### PR TITLE
Update readme and cleanup structure of repo

### PR DESCRIPTION
**Pull request by alexanderattar**
## Describe Changes

- Update README
- Add CONTRIBUTIONS
- Move `externalDeployments` directory into sub-directory `deployments/external`
- Update hardhat config for the above

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] All Hardhat tests are passing

### CODE REVIEW

**Comment by ACC01ADE** [comment]
Had to do some minor cleanup since `deployments` dir was broken and not working according to the multiple environments logic. Also `config` dir still required some files in there, for it to not break and work well. Updated the `hardhat-holograph-contract-builder` plugin to work with minimal `config` dir files.
Repo included in itself the draft `feature/operator-pod-logic` branch, had to revert some changes to properly reflect smart contract logic to what it currently being used on-chain.

**Comment by ACC01ADE** [comment]
Tests now work and pass successfully. Smart contracts are an exact version of what is on-chain being used by public testnet users.

**Review by ACC01ADE — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,59 +1,60 @@
...
-
-**This project uses [asdf](https://asdf-vm.com/) for tool versions management.**
-
----
-
-## First-run
-
-If this is your first time running the project. Some initial steps will need to be taken to prepare the project.
+<div align="center">
+  <a href="https://holograph.xyz" target="_blank"><img alt="Holograph" src="https://user-images.githubusercontent.com/21043504/188220186-9c7f55e0-143a-41b4-a6b8-90e8bd54bfd9.png" width=600></a>
```
Comment (ACC01ADE) [inline README.md]: Unfortunately, GitHub does not allow for `target` tag in links. Will need to revert back to previous version.

Comment (ACC01ADE) [reply README.md]: Reverted to previous commit.

**Review by ACC01ADE — APPROVED** [review]

**Comment by sogoiii** [comment]
I think we can add a few of these tags like these repos:
image
image

We can probably add the following:
1. Lint 
2. Test
3. Build
4. Discord
5. Twitter
6. yarn
7. License

**Comment by sogoiii** [comment]
We can also clarify the asdf section. We can communicate to the user how to install the required plugins so that `asdf` works as expected. Maybe something like
```
...
2. Run asdf install to have the correct tool versions. Install the following plugins before moving forward
  - Install [asdf Node plugin](https://github.com/asdf-vm/asdf-nodejs): `asdf plugin add nodejs https://github.com/asdf-vm/asdf-nodejs.git`
  - Install [asdf yarn plugin](https://github.com/twuni/asdf-yarn): `asdf plugin-add yarn`
...
```

**Comment by sogoiii** [comment]
I would also reorder the sections and maybe add table of contents section.

1. Purpose - here have the main description and a link to other stuff like the docs and community
3. Table of contents  (maybe?)
4. Development 
5. Branching Model and Releases
6. Community
7. Contributing
8. License

**Comment by alexanderattar** [comment]
> I think we can add a few of these tags like these repos: image image
> 
> We can probably add the following:
> 
> 1. Lint
> 2. Test
> 3. Build
> 4. Discord
> 5. Twitter
> 6. yarn
> 7. License

Yeah the tags would be nice. Need to setup Github actions for the tests

**Pull request merged**